tests/run-make/windows-ws2_32/rmake.rs

```diff
@@ -0,0 +1,14 @@
+//@ only-msvc
+
+// Tests that WS2_32.dll is not unnecessarily linked, see issue #85441
+
+use run_make_support::{llvm_readobj, rustc, tmp_dir};
+
+fn main() {
+    rustc().input("empty.rs").run();
+    let empty = tmp_dir().join("empty.exe");
+    let output = llvm_readobj().input(empty).coff_imports().command_output();
+    assert!(output.status.success());
```

Comment:
Reminder for myself: cc https://github.com/rust-lang/rust/issues/125617 we should add run() that asserts success for llvm-readobj as well, so it's less of a footgun to forgor checking cmd status

Reply:
Oh, wait there is a run I think. I missed that.

Reply:
Yeah, I think we probably need to un-provide the `command_output` helper because it's probably almost never the right helper to use

Reply:
Sounds helpful. I should probably also read the docs for the support lib.

Reply:
Oh, that gives me a thought: it might be nice to make `./x doc run-make-support --open` work.

Reply:
Yes, that would indeed be very useful